src/cmd_line.rs

```diff
@@ -101,6 +109,18 @@ pub fn handle_command_line_arguments() -> Result<(), String> {
                 .takes_value(true)
                 .last(true)
                 .help("Specify Arguments to pass down to neovim"),
+        )
+        .arg(
+            Arg::with_name("wayland_app_id")
+                .long("appid")
+                .short("a")
+                .takes_value(true)
+        )
+        .arg(
+            Arg::with_name("x11_class")
+                .long("class")
```

Comment:
```suggestion
                .long("x11-wm-class")
```

I'd sugges using this, and dropping short argument too.